### PR TITLE
Add Franklin OH orthoimagery

### PR DESCRIPTION
**Pull request by cicku**
Public domain.

### CODE REVIEW

File: sources/north-america/us/oh/Franklin_OH_2019.geojson [hunk 1]
```diff
@@ -0,0 +1,280 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "Franklin_OH_2019",
+        "attribution": {
+            "url": "https://auditor-fca.opendata.arcgis.com/",
```
Comment (1ec5) [inline sources/north-america/us/oh/Franklin_OH_2019.geojson:6]: #616 suggests an icon for attribution, not required though.

Comment (cicku) [reply sources/north-america/us/oh/Franklin_OH_2019.geojson:6]: That logo in #616 appears to be off-topic with Franklin County...

The one used here is perfect.  
image

Comment (cicku) [reply sources/north-america/us/oh/Franklin_OH_2019.geojson:6]: https://github.com/osmlab/editor-layer-index/commit/e4232ebda55d59b9a8a05e08de0d9b42c91f4af3

Comment (1ec5) [reply sources/north-america/us/oh/Franklin_OH_2019.geojson:6]: Thanks, that makes sense. The same imagery is available from the City of Columbus, hence the confusion.

**Pull request merged**